planet/management/commands/update_planet.py

```diff
@@ -0,0 +1,139 @@
+"""
+update_planet
+
+Imports all feeds for users who have filled in a valid website and website_rss,
+the amount of items imported is limited to the defined FEED_LIMT
+
+Usage: ./manage.py update_planet
+"""
+
+
+import logging
+import sys
+import time
+
+from datetime import datetime
+from pytz import utc
+
+import bleach
+import feedparser
+
+from django.core.cache import cache
+from django.core.management.base import BaseCommand
+from django.template.defaultfilters import truncatewords_html
+from django.conf import settings
+
+from planet.models import Feed, FeedItem, FEEDITEM_SUMMARY_LIMIT
+
+
+logging.basicConfig(
+    level=logging.WARNING,
+    format=u'%(asctime)s -> %(levelname)s: %(message)s',
+    datefmt=u'%Y-%m-%d %H:%M:%S',
+    stream=sys.stderr)
+logger = logging.getLogger()
+
+
+class Command(BaseCommand):
+
+    def parse_feed(self, feedmodel):
```

Comment:
`feed` would feel nicer, since you're passing in an instance rather than the model class itself.